HackingWithSwift/SwiftUI/Project1/MyProject/MyProject/ContentView.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  ContentView.swift
+//  MyProject
+//
+//  Created by Katasee on 07/10/2024.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    @State private var currencyToExchange: String = "USD"
+    @State private var currencyExchanged: String = "EUR"
+    @State private var amount: Double = 0
+    
+    @State private var convertToAmount: Double = 0
+    var currency: [String] = ["USD", "EUR", "PLN"]
+    
+    let usd: [String: Double] = ["USD": 1.0, "EUR": 1.1, "PLN": 4.3]
+    let eur: [String: Double] = ["USD": 0.91, "EUR": 1.0, "PLN": 3.9]
+    let pln: [String: Double] = ["USD": 0.26, "EUR": 0.23, "PLN": 1.0]
+    
+    var body: some View {
+        
+        NavigationStack {
+            ZStack{
+                Color(.secondarySystemBackground)
+                    .ignoresSafeArea()
+                ScrollView {
+                    VStack{
+                        VStack {
+                            titleOfTextField()
+                            buildTextField()
+                        }
+                        
+                        VStack (spacing: 1) {
+                            firstPicker
+                            Divider()
+                            secondPicker
+                        }
+                        
+                        VStack {
+                            resaultTitle
+                            resault
+                        }
+                    }
+                    .padding(.horizontal, 24)
+                }
+                .navigationTitle("Exchange 24")
+            }
+        }
+    }
+    
+    func titleOfTextField() -> some View {
+        Text("Curreency convert")
+            .font(.subheadline)
+            .foregroundStyle(Color.gray)
+            .frame(width: 350, height: 15,alignment: .leading)
+    }
+    
+    func buildTextField() -> some View {
+        TextField("Enter amout", value: $amount, format: .currency(code: currencyToExchange))
```

Comment:
It's not a big deal now, however when it comes to commercial code, you have to ensure that you have no typos in your code.